benches/api_calls.rs

```diff
@@ -309,6 +309,28 @@ mod tests {
         });
     }
 
+    #[bench]
+    fn jni_get_string(b: &mut Bencher) {
+        let env = VM.attach_current_thread().unwrap();
+        let string = env.new_string("test").unwrap();
```

Comment:
Will be nice to put this into a constant, for referencing / robustness.

Also, using a longer string with UTF-8 chars in it would be a good idea.